src/views/emote-upload/EmoteUpload.vue

```diff
@@ -72,23 +89,15 @@
 									<Icon v-else icon="times" color="red" />
 								</div>
 							</div>
-						</div>
-						<input id="file-upload" hidden type="file" :accept="mimeList" @change="onFileInputChange" />
-						<label for="file-upload">
-							<img ref="previewImage" />
-						</label>
-					</div>
-
-					<span>
-						<div v-if="parentEmote" class="parent-emote">
-							<img :src="getImage(parentEmote.host, ImageFormat.WEBP, 2)?.url" />
-							<div class="as-child-notice">
-								<i18n-t keypath="emote.upload.as_child" tag="p">
-									<span style="font-weight: 600">{{ parentEmote.name }}</span>
-								</i18n-t>
+							<div class="requirements">
+								<h3>Requirements:</h3>
+								<br />- under 7 MB <br />- smaller than 1000 x 1000 pixels <br />- less than 1000 frames
```

Comment:
Do not use legacy tags such as "`<br />`", use css or paragraph tags instead.

Reply:
can i know why legacy tags should not be used?

Reply:
because they're legacy and this is not 2008